src/pkgdev/cli.py

```diff
@@ -76,7 +75,7 @@ def parse_config_sections(self, namespace, sections):
 
     def parse_config_options(self, namespace, configs=()):
         """Parse options from config if they exist."""
-        configs = [x for x in configs if os.path.isfile(x)]
```

Comment:
As a suggestion, I think keeping all subcommand options in the same overarching config file (e.g. pkgdev.conf) is a better way to go, using separate config sections per subcommand (similar to git config support).

It looks like this is trending towards splitting it all up into separate files per subcommand.

Reply:
Yeah. I was getting too much setbacks for unknown arguments passed to argparser and everything blowing up (as my first commit in this PR).

I still prefer just one config file, but I'm unable to understand how snakeoil's file parsing works under the hood. I'm also not sure what people will do when a "conflict" arises (for example `--ask` for commit & push) and they want separate default.

I still have an idea of maybe section dividing (like `[DEFAULT.commit]`, `[gentoo.push]`) but not sure how much deeper I need to go.